client/src/App.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-import PlayersListTable from "./components/PlayerListTable/PlayerListTable";
-import ClubList from "./components/ClubList/ClubList";
+import PlayersListTable from "./components/PlayersList/PlayersList";
+import ClubsList from "./components/ClubsList/ClubsList";
 import NavBar from "./components/NavBar/NavBar";
 import { Routes, Route } from "react-router-dom";
+import ClubPage from "./components/ClubPage/ClubPage";
 
 const App = () => (
   <>
     <NavBar />
     <Routes>
       <Route path="/" element={<PlayersListTable />} />
-      <Route path="/teams" element={<ClubList />} />
+      <Route path="/teams" element={<ClubsList />} />
+      <Route path="/teams/:id" element={<ClubPage />} />
```

Comment:
```suggestion
      <Route path="/team/:id" element={<ClubPage />} />
```
and anywhere it's being used